core/Loc.cc

```diff
@@ -211,7 +211,10 @@ pair<Loc, u4> Loc::findStartOfLine(const GlobalState &gs) const {
     u4 lineStart = Loc::pos2Offset(this->file().data(gs), {startDetail.line, 1});
     std::string_view lineView = this->file().data(gs).source().substr(lineStart);
 
-    u4 padding = lineView.find_first_not_of(" \t");
+    size_t padding = lineView.find_first_not_of(" \t");
+    if (padding == string::npos) {
+        padding = 0;
```

Comment:
could you leave a comment on why `0` is the right choice here?
alternatively, If there is no start of the line, maybe this method should return `std::nullopt`?

Reply:
I think it makes sense to just set `padding = 0` here (and not return `std::nullopt`). I left a comment.